src/panels/config/devices/ha-config-devices-dashboard.ts

```diff
@@ -235,37 +239,51 @@ export class HaConfigDeviceDashboard extends LitElement {
 
   private _columns = memoizeOne(
     (narrow: boolean, showDisabled: boolean): DataTableColumnContainer => {
-      const columns: DataTableColumnContainer = narrow
-        ? {
-            name: {
-              title: this.hass.localize(
-                "ui.panel.config.devices.data_table.device"
-              ),
-              main: true,
-              sortable: true,
-              filterable: true,
-              direction: "asc",
-              grows: true,
-              template: (name, device: DataTableRowData) => html`
-                ${name}
-                <div class="secondary">
-                  ${device.area} | ${device.integration}
-                </div>
-              `,
-            },
-          }
-        : {
-            name: {
-              title: this.hass.localize(
-                "ui.panel.config.devices.data_table.device"
-              ),
-              main: true,
-              sortable: true,
-              filterable: true,
-              grows: true,
-              direction: "asc",
-            },
-          };
+      const columns: DataTableColumnContainer = {
+        icon: {
+          title: "",
+          type: "icon",
+          template: (_icon, device) =>
+            device.domains.length
+              ? html`<img
+                  referrerpolicy="no-referrer"
+                  src=${brandsUrl({
+                    domain: device.domains[0],
+                    type: "icon",
+                    darkOptimized: this.hass.themes?.darkMode,
+                  })}
+                />`
```

Comment:
Let's not make #14178 worse.  I cannot see your screenshot; is this sitting next to the device name or in place of it?

Reply:
It sits next to it (in front of it), but it adds no value for screen readers I think, it is just a logo of the brand. I think we should just set `aria-hidden` the image in this case.

Reply:
yeah, it is really the logo/icon of the manufacturer/product. So if it would have an `alt`, it would be the integration name. The integration name is also already in the table, so a bit redundant in that case I would say.

Reply:
If it's an icon to represent the manufacturer or integration, then shouldn't it be in one of those columns?  In that case, yes `alt=""` is appropriate.

Otherwise it's adding extra information to the device name column that isn't already there, and seems misplaced since those have columns of their own.